README.md

```diff
@@ -1,15 +1,25 @@
 # Lumina
 
-Rust implementation of Celestia's [data availability node](https://github.com/celestiaorg/celestia-node) able to run natively and in browser-based environments.
+Rust implementation of Celestia's [data availability node](https://github.com/celestiaorg/celestia-node) able to run natively and in browser-based environments. 
+
+Run Lumina now at [lumina.rs](https://lumina.rs/) and directly verify Celestia.
 
 Supported features:
-- [x] Synchronize and verify `ExtendedHeader`s from genesis to the network head
-- [x] Header exchange (`header-ex`) client and server
-- [x] Listening for, verifying and redistributing extended headers on gossip protocol (`header-sub`)
-- [x] Persistent store for Headers
-- [x] Integration tests with Go implementation
-- [ ] Data Availability Sampling
-- [ ] Creating, distributing, and listening for Fraud proofs
+- Backward and forward synchronization of block headers within syncing window
+- Header exchange (`header-ex`) client and server
+- Listening for, verifying and redistributing extended headers on gossip protocol (`header-sub`)
+- Listening for, verifying and redistributing fraud proofs on gossip protocol (`fraud-sub`)
+- Backward and forward Data Availability Sampling
+- Native and browser persistent storage
+- Streaming events happening on the node
+- Native and wasm library, embed the node anywhere
+- Integration tests with Go implementation
+
+> [!NOTE]
+> Lumina implements [`shwap`](https://github.com/celestiaorg/CIPs/blob/main/cips/cip-19.md) protocol to perform DASing,
+which is not yet enabled on all networks in the Go implementation. This means that even tho Lumina will be sampling all
+blocks, the network is unlikely to provide requested data yet. Shwap is going to become the main DASing protocol in next
+celestia-node updates.
```

Comment:
```suggestion
blocks, the network is unlikely to provide the requested data yet. Shwap is going to become the main DASing protocol in 
the upcoming celestia-node versions.
```
Trying to strike a balance between being non-commital and it sounding good. wdyt?